app/src/main/java/com/hover/stax/utils/Utils.kt

```diff
@@ -246,6 +245,17 @@ object Utils {
         }
     }
 
+    fun getActivity(context: Context): Activity? {
```

Comment:
If you find yourself needing to do this, then you've complicated your solution. Context should be injected, inferred or passed in most cases.